server/api/logger/main.js

```diff
@@ -1,26 +1,23 @@
+import _ from "lodash";
 import bunyan from "bunyan";
 import bunyanFormat from "bunyan-format";
-import Bunyan2Loggly from "bunyan-loggly";
-import { includes } from "lodash";
+import { Meteor } from "meteor/meteor";
+import Bunyan2Loggly from "./loggly";
 
 // configure bunyan logging module for reaction server
 // See: https://github.com/trentm/node-bunyan#levels
 const levels = ["FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE"];
 
 // set stdout log level
 let level = process.env.REACTION_LOG_LEVEL || Meteor.settings.REACTION_LOG_LEVEL || "INFO";
-let outputMode = "short";
+const outputMode = "short";
 
 level = level.toUpperCase();
 
-if (!includes(levels, level)) {
+if (!_.includes(levels, level)) {
   level = "INFO";
 }
 
-if (level === "TRACE") {
```

Comment:
Well, my thinking here was that the default stdout output stream (the console) is intended entirely for human consumption, so readability should be the main focus of the stdout configuration.  Also, there isn't really anything that an unformatted output gives you that's unique or useful.  The only things the formatter strips out is the hostname and pid (process ID).  Any other objects (errors, data, etc.) passed to the logger calls for debugging are still included in the formatted log output.  The main advantage being that they're much easier to read when formatted like that.  So in short, all of the same data gets shown, it's just easier to parse visually when it's run through the formatter.

However, in case someone _really_ wants to have that raw JSON output, I added a new env var (`REACTION_LOG_FORMAT`) which you can use to set the formatter.  So...

```
REACTION_LOG_FORMAT=json reaction
```
outputs this...

<img width="1293" alt="1____code_meteor_reaction_reaction__node__and_main_js_ ___code_meteor_reaction_reaction" src="https://user-images.githubusercontent.com/3077956/27237561-8c8a101a-5297-11e7-987f-43a5cec1be54.png">

And technically you could set that to anything [`bunyan-format`](https://www.npmjs.com/package/bunyan-format) supports.  Which is...

`short|long|simple|json|bunyan` (default: short)

Sort of relevant... there's not a single instance of `Logger.trace()` in the whole app, so this shouldn't have any effect on anyone's current experience.